controllers/limitador_controller.go

```diff
@@ -89,6 +96,26 @@ func (r *LimitadorReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Complete(r)
 }
 
+func (r *LimitadorReconciler) reconcileStatus(ctx context.Context, limitadorObj *limitadorv1alpha1.Limitador) error {
+	logger := logr.FromContext(ctx)
+	// Simple enough now, we could implement a thorough check like with RateLimit with ObservedGeneration in the future
+	builtServiceUrl := buildServiceUrl(limitadorObj)
+	if builtServiceUrl != limitadorObj.Status.ServiceURL {
+		logger.V(1).Info("Updating the Status", "Name", limitadorObj.Name)
+		limitadorObj.Status.ServiceURL = builtServiceUrl
+		return r.Client().Status().Update(ctx, limitadorObj)
+
+	}
+	return nil
+}
+
+func buildServiceUrl(limitadorObj *limitadorv1alpha1.Limitador) string {
```

Comment:
Now it creates a single URL for the HTTP port, since it was the one required by the Rate Limits... Would it make sense to craft separate URLs with both ports?